C/regression4.h

```diff
@@ -0,0 +1,19 @@
+#ifndef SIMPLICITY_REGRESSION4_H
+#define SIMPLICITY_REGRESSION4_H
+
+#include <stddef.h>
+#include <stdint.h>
+#include "bounded.h"
+
+/* A length-prefixed encoding of the following Simplicity program:
+ *     uWitness OneV : f 15 ++ [uComp (3*2^16) 1]
```

Comment:
This looks like the program in the OP of https://github.com/BlockstreamResearch/simplicity/issues/209, but with an additional `++ [uComp (3*2^16) 1]`?

Reply:
the last uComp composes the main program with `uWitness`.

Here the `:` is list cons in Haskell, which always confuses me when I read this comment to myself.